internal/processor/filterconfig/config.go

```diff
@@ -1,53 +1,56 @@
-// Copyright The OpenTelemetry Authors
+// Copyright  The OpenTelemetry Authors
 //
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//       http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Why this changes?

Reply:
That is how the license generator made it.